modules/linear_boltzmann_solvers/discrete_ordinates_curvilinear_solver/sweep_chunks/lbs_curvilinear_sweep_chunk_pwl.cc

```diff
@@ -66,7 +66,7 @@ SweepChunkPwlrz::SweepChunkPwlrz(
       map_polar_level_.emplace(dir_idx, dir_set.first);
 
   //  set normal vector for symmetric boundary condition
-  const int d = (grid_.Attributes() & DIMENSION_1) ? 2 : 0;
+  const int d = (grid_.Dimension() == 1) ? 2 : 0;
   normal_vector_boundary_ = Vector3(0.0, 0.0, 0.0);
   normal_vector_boundary_(d) = 1;
```

Comment:
I do not fully follow this. When dim=1, th normal is [0,0,1], otherwise it is [1,0,0]. Why [1,0,0] for dim=2/3 ?

Reply:
I think it must have something to do with the fact that the 1D orthogonal mesh generator, creates the mesh in the z-coordinates https://github.com/Open-Sn/opensn/blob/0f4fee4410b6379fab86a4b9367d33d026a55241/framework/mesh/mesh_generator/orthogonal_mesh_generator.cc#L121-L125

We should change that so it is in x? If so, I can open an issue for that task.

Reply:
this file is for the cylindrical coordinate system (given its name `lbs_curvilinear....`). I am not even sure what 1D should mean there, given what the lines 68-71 do. Tagging @zhardy-lanl  and @wdhawkins 
1. in 1D cylindrical, the coordinate system should be \vec{e_r} (the radial dimension). The normal should be [1,0,0] as I expect the radial coordinate to be the first one. What those lines above do is treat 1D cylindrical like 1D Cartesian and I agree that in 1D Cartesian, we have selected z to be the axis and not x. In cylindrical coordinates, the polar angle is really with respect to the cylindrical axis.  Implementation wise, I am still confused why 1D cyl coord has a normal as [0,0,1]. The comments states symmetric boundary condition, and indeed, the axis of the cylinder is a symmetric BC. 
2. In multi-D cylindrical, I *do* understand the choice of normal to be [1,0,0] as it fits what I described above. 
3. So, I must conclude that in the starting direction case of cylindrical, which is a 1D slab simulation, the code calls the 1D solver of Cartesian coordinates, which has z as its only axis hence the normal [0,0,1] and for the other sweeping directions, i.e., in true cylindrical sweeps, the normal is set to [1,0,0]. 
I will wait for others to comment. My explanation or understanding is too convoluted. I hope I am not right as this would be a hacky implementation

Reply:
If I am reading it correctly, `DiscreteOrdinatesCurvilinearSolver::PerformInputChecks()` has a check that only allows 2D cylindrical.

Reply:
@zhardy-lanl @andrsd I've updated the curvilinear issue to include this to check.